client/src/components/ElementEditor/Element.js

```diff
@@ -11,15 +12,19 @@ class Element extends PureComponent {
     const {
       element: { ID, Title, BlockSchema },
       HeaderComponent,
-      ContentComponent
+      ContentComponent,
+      link,
     } = this.props;
+    const linkTitle = i18n.inject(
+      i18n._t('ElementalElement.TITLE', 'Edit this {type} block'), { type: BlockSchema.type }
+    );
 
     if (!ID) {
       return null;
     }
 
     return (
-      <div className="element-editor__element">
+      <a className="element-editor__element" href={link} title={linkTitle}>
```

Comment:
Hmm, I don't know if I like this. Can you nest the anchor inside the div instead?

Reply:
I will have to remove all padding from the div and add it to the anchor - which is what that class did - so it made sense to move the class to the anchor tag, then the div was kind of pointless. Not really sure what a better solution to that would be?

Reply:
Ok, I'll get over it